scripts/partition_manager.py

```diff
@@ -357,39 +298,26 @@ def parse_args():
 
 "pm_config.h" is in the same folder as the given 'pm.yml' file.''',
         formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument("-i", "--input", required=True, type=str, nargs="+",
-                        help="Space separated list of configs."
-                             "Each config is a ':' - separated list of the following properties:"
-                             "image-name:pm.yml-path:build_dir:out_dir")
-    parser.add_argument("--app-pm-config-dir", required=True,
-                        help="Where to store the 'pm_config.h' of the root app.")
-    parser.add_argument("--app-build-dir", required=True,
-                        help="Path to of the root app's build directory.")
 
-    return parser.parse_args()
+    parser.add_argument("-n", "--input-names", required=True, type=str, nargs="+",
```

Comment:
To be pep-8 compliant, ensure the `help=` is vertically aligned with the "-n", ie the column after the opening `(`

for all arguments.

Reply:
Done